src/main/webapp/js/instructorCourseEnrollPage.es6

```diff
@@ -1,20 +1,20 @@
-'use strict';
+/* global BootboxWrapper:false StatusType:false */
 
-var loadUpFunction = function() {
-    var typingErrMsg = 'Please use | character ( shift+\\ ) to seperate fields, or copy from your existing spreadsheet.';
-    var notified = false;
+const loadUpFunction = function () {
+    const typingErrMsg = 'Please use | character ( shift+\\ ) to seperate fields, or copy from your existing spreadsheet.';
+    let notified = false;
 
     function isUserTyping(str) {
         return str.indexOf('\t') === -1 && str.indexOf('|') === -1;
     }
```

Comment:
Notes for a future PR, some parentheses can help readability here. I did a double take when looking at this. :stuck_out_tongue: